src/modules/app/components/navigation/navigation.component.scss

```diff
@@ -32,8 +32,8 @@
 .logout-button {
   margin-top: 30px;
   margin-bottom: 30px;
-  padding-right: 16px; 
-  margin-left: auto
+  margin-left: 240px;
```

Comment:
Bitte ohne hardcoded Pixel values. Mit ner flexbox geht das ohne, dass es zusammenbricht, wenn sich die Breite der Navbar aendert.

Reply:
Habe leider keine elegante Möglichkeit gefunden gestern. Im Template steht der Button in dem selben mat-list-item wie der Username und das Logo, und die werden auch gleichmäßig verteilt, aber irgendwie wird der Platz in der Zeile nie richtig ausgenutzt und ist eher links-bündig. https://github.com/agri-gaia/DataDashboard/blob/main/src/modules/app/components/navigation/navigation.component.html#L19